lib/Utils/CodeParser.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace TwentySixB\Translations\Utils;
+
+use KKomelin\TranslatableStringExporter\Core\CodeParser as CoreCodeParser;
+
+/**
+ * Code Parser, used for making pots in Laravel.
```

Comment:
```suggestion
 * Code Parser, used for generating pots in Laravel.
```

Reply:
Shouldn't it be used for "extracting string from..." ?

Reply:
They're both correct but I'll use the "extracting strings from" one 😁